docs-v2/integrations/all/twenty-crm.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Twenty CRM
+sidebarTitle: Twenty CRM
+---
+
+API configuration: [`twenty-crm`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (API Key)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+-   [API docs](https://app.twenty.com/settings/developers)
+-   [Generate API keys](https://app.twenty.com/settings/developers)
+-   [Authentication](https://twenty.com/developers/rest-api/core#/)
+-   [API docs](https://app.twenty.com/settings/developers)
```

Comment:
It looks like you may have included a duplicate link to the docs, I'm not sure if that was intentional. Additionally, you could consider adding a link to the documentation on rate limiting.

Reply:
the duplicate link has been removed